supervisor/config.py

```diff
@@ -118,6 +119,16 @@ def version(self, value: AwesomeVersion) -> None:
         """Set supervisor version."""
         self._data[ATTR_VERSION] = value
 
+    @property
+    def enable_ipv6(self) -> bool | None:
+        """Return IPv6 configuration for docker network."""
+        return self._data.get(ATTR_ENABLE_IPV6)
+
+    @enable_ipv6.setter
+    def enable_ipv6(self, value: bool) -> None:
+        """Set IPv6 configuration for docker network."""
+        self._data[ATTR_ENABLE_IPV6] = value
+
```

Comment:
Since we expose the Docker IPv6 config in the `/docker/options` endpoint I'd also store the config in the Docker configuration file (`DockerConfig` class, extend `SCHEMA_DOCKER_CONFIG`).

Reply:
I meant to really move these two properties in `DockerConfig`. The property would then be accessible using `self.sys_docker.enable_ipv6` (instead of `self.sys_config.enable_ipv6`).